common/djangoapps/enrollment/data.py

```diff
@@ -93,6 +93,31 @@ def get_course_enrollment(username, course_id):
         return None
 
 
+def get_user_enrollments(course_id, serialize=True):
+    """Based on the course id, return all user enrollments in the course
+    Args:
+        course_id (str): The course to retrieve enrollment information for.
+        serialize (bool): Boolean denoting whether to serialize the enrollments
```

Comment:
Per YAGNI, no need to add a `serialize` parameter.
Also, we don't want to have an API with a default parameter value that can potentially cause a bad performance issue - returning _all_ enrollments within a course.